flink-connectors/flink-connector-cassandra/pom.xml

```diff
@@ -65,6 +68,7 @@ under the License.
 							<goal>shade</goal>
 						</goals>
 						<configuration combine.self="override">
+							<shadedArtifactAttached>${shade.shadedArtifactAttached}</shadedArtifactAttached>
```

Comment:
Why can't we define this in the parent pom globally for all shade plugin instances?